cloudformation/build/deployment-pipeline.yaml

```diff
@@ -363,7 +363,7 @@ Resources:
                       "PublicDomainName": "${PublicDomainName}",
                       "Repository": "${Repository}",
                       "ApplicationName": "${ApplicationName}",
-                      "Prefix": "${Prefix}",
+                      "Subdomain": "${Subdomain}",
                       "ContainerPort": "${ContainerPort}",
```

Comment:
ApplicationName (defaulted to REPO) and Subdomain will often be the same, but not necessarily always, so keeping both (?).